networkx/algorithms/bipartite/generators.py

```diff
@@ -432,6 +438,7 @@ def preferential_attachment_graph(aseq, p, create_using=None, seed=None):
 
 
 @py_random_state(3)
+@nx._dispatch(graphs=None)
 def random_graph(n, m, p, seed=None, directed=False):
```

Comment:
Should we use the dispatch name of `"bipartite_random_graph"` here? Should we be careful with any other names in `bipartite` submodule?

Reply:
Yes -- names in the bipartite subpackage can clobber names in the main namespace. So we should handle bipartitie in a similar way to how It was handled for some algorithms like `approximations`.

Reply:
Yeah, this PR currently changes the name of those that are clobbered, and so far our approach has been to _not_ preemptively add e.g. `"bipartite_"` or `"approximate_"` prefix. Following current precedent, we don't add `"bipartite_"` here even though `random_graph` seems kind of generic.

Reply:
oh wait, you answered "yes"... should we add `"bipartite_"` prefix?

Reply:
You should treat it the same as you did for the algorithms. If you only did it for the names that actually clobber, then let's stick to that here too.
:)